app/controllers/providers/proceeding_merits_task/opponents_application_controller.rb

```diff
@@ -0,0 +1,34 @@
+module Providers
+  module ProceedingMeritsTask
+    class OpponentsApplicationController < ProviderBaseController
```

Comment:
It might seem like a trifling issue, but I think we should be keeping our naming conventions as conventional as possible.

I _think_ that would mean an `OpponentsApplication` record backed by an `opponents_applications` table with an `OpponentsApplicationsController` by default.

Making sure we name things (and pluralise them) intentionally by convention should help us keep things simple (like our routes).

This is one of those things I normally need to double check myself, but I think it's worthwhile us trying to minimise these deviations from convention where we can!

Reply:
i had started with a plural controller name but changed to singular as the route and relations are all singular and convention only [favours pluralisation](https://guides.rubyonrails.org/action_controller_overview.html#controller-naming-convention) but does not adhere to it itself (i.e. ApplicationController).

I would approve either way